charts/brigade-cron-event-source/templates/cronjob.yaml

```diff
@@ -35,10 +35,6 @@ spec:
               {{- toYaml $.Values.securityContext | nindent 14 }}
             image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag | default $.Chart.AppVersion }}"
             imagePullPolicy: {{ default "IfNotPresent" $.Values.image.pullPolicy }}
-            {{- with $.Values.resources }}             
```

Comment:
Can we comment that out in Values file and just set some default values here?

Reply:
Although it's a best practice in k8s to specify resource requirements, it has oddly never been an encouraged practice to have default resource requirements in a Helm chart. In fact, a newly-generated Helm chart has this in the `values.yaml` file:

```yaml
resources: {}
  # We usually recommend not to specify default resources and to leave this as a conscious
  # choice for the user. This also increases chances charts run on environments with little
  # resources, such as Minikube. If you do want to specify resources, uncomment the following
  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
  # limits:
  #   cpu: 100m
  #   memory: 128Mi
  # requests:
  #   cpu: 100m
  #   memory: 128Mi
```

I'd like to observe that establishes practice by not having any default resource limits/requirements.

Beyond not having a default, I can't see any utility in being able to configure this. Realistically, this program is so small that each of these pods will run for less than a second and use almost no resources.